### PR TITLE
Windows: add a roadmap page

### PR DESCRIPTION
**Pull request by laszlocsomor**

### CODE REVIEW

**Comment by laszlocsomor** [comment]
ping @spomorski , @dslomov , @davidstanke , @ulfjack ?

**Comment by laszlocsomor** [comment]
Thanks @ulfjack !

**Comment by laszlocsomor** [comment]
ping @spomorski , @davidstanke

**Comment by spomorski** [comment]
Let me look today.

File: roadmaps/windows.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        `//src/test/shell/...`
+        ([#4292](https://github.com/bazelbuild/bazel/issues/4292)).
+    *   Run 90% of `//src/...` on Windows
+        ([#4292](https://github.com/bazelbuild/bazel/issues/4292)).
+    *   Fix `$(location)` expansion in `*_test.args`
+        ([#4171](https://github.com/bazelbuild/bazel/issues/4171)).
+    *   Speed up bootstrap test
+        ([#4503](https://github.com/bazelbuild/bazel/issues/4503)).
+
+*   Fix Bazel client bugs.
```
Comment (spomorski) [inline roadmaps/windows.md]: I would bold the major bullets for better scannability.

**Pull request merged**